postgres/init.sls

```diff
@@ -15,7 +15,7 @@ install-postgresql-repo:
 install-postgresql:
   pkg.installed:
     - name: {{ postgres.pkg }}
-    - refresh: {{ 'use_upstream_repo' in pillar.get('postgres') }}
+    - refresh: {{ salt['pillar.get']('postgres:use_upstream_repo') }}
```

Comment:
Test it to double check, but I'm pretty sure the default returned by pillar.get evaluates to False (at least enough for an if test that is).